src/vec_int512_runtime.c

```diff
@@ -746,7 +748,9 @@ __VEC_PWR_IMP (vec_mul512_byMN) (__VEC_U_512 *p,
   __VEC_U_512 *np = m2;
   unsigned long mx = M;
   unsigned long nx = N;
+#if __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
   unsigned long px = M + N;
```

Comment:
where is this used?

Reply:
In the (__PDX) quadword index macro for BE

#define __PDX(__index) ((px - 1) - (__index))

This is a catch-22 for lint/-Wall. px is used in BE but is unused in LE. 

#define __PDX(__index) (__index)